examples/android/app/src/main/jni/CMakeLists.txt

```
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 3.22)
+project(sfml-example)
+
+set(SFML_DIR "${CMAKE_ANDROID_NDK}/sources/third_party/sfml/lib/${CMAKE_ANDROID_ARCH_ABI}/cmake/SFML")
+
+# Create the libapp.so library that contains the application's c++ code
+add_library(app SHARED)
+target_sources(app PRIVATE main.cpp)
+
+# Find and link SFML.
+# We must link to the network and audio components even if we don't use them, because sfml-activity will try to load them
+find_package(SFML 3 REQUIRED COMPONENTS Graphics Audio Network CONFIG)
```

```suggestion
find_package(SFML 3 REQUIRED COMPONENTS Graphics Audio Network)
```

It's unlikely that anyone uses a fine module to find SFML 3 but it's not invalid to do so let's not require using a config module.